docs/Account sign up - Approved and rejected sign up flows.md

```diff
@@ -0,0 +1,148 @@
+
+## Helpful links
+- [Calling Dynamics 365 Fraud Protection](./Authenticate&#32;and&#32;call&#32;Fraud&#32;Protection.md)
+- [SignUp - Data model and endpoint](https://apidocs.microsoft.com/services/dynamics365fraudprotection#/v1.0/V1.0MerchantservicesEventsSignUpPost)
+- [SignUp Status - Data model and endpoint](https://apidocs.microsoft.com/services/dynamics365fraudprotection#/v1.0/V1.0MerchantservicesEventsSignUpStatusPost)
+- [Sample site - Register a user](../src/Web/Controllers/AccountController.cs) (see Register POST method)
+- [Sample site - Dynamics 365 Fraud Protection service](../src/Infrastructure/Services/FraudProtectionService.cs) (see PostSignUp and PostSignUpStatus methods)
+
+## Sign up flow
+The sign up event flow consists of:
+1. Sign up event
+1. Sign up status event
+
+**NOTES**
+- You are not required to send these events, much less send them in any particular order, or within any specific time. The sample site does, in fact, follow this order. 
+- As the merchant, you decide when to call the SignUp API. It may be before or after you have created an account in your database. The sample site calls the SignUp API before it creates a user in its database and only creates the user in the database if it decides to approve the sign up. Notice how this sample site does not send a user ID until the sign up status event, and even then, only when it decides to approve the user. This implementation decision is guided by the sample site's behavior.
+- Set the 'assessmentType' field based on if you plan to use the Dynamics 365 Fraud Protection risk score:
+  - Pass 'Evaluate' if you do NOT plan to use the risk score and are still evaluating Dynamics 365 Fraud Protection against your existing fraud solution.
+  - Pass 'Protect' if you plan to use the risk score . This creates more accurate reports when we can distinguish between your 'Evaluate' and 'Protect' API calls.
+- In the sample site, an existing user's ID is set to their email address which is also tied to how they log in to the sample site. Furthermore, the sample site ensures that no two customers have the same email address. This ensures user IDs will be unique for the sample site, as well as unique when sent to Dynamics 365 Fraud Protection. Decide what format to use for existing user IDs. It does not have to be an email address, but you should be careful not to duplicate user IDs.
+
+## Required data
+- Sign up ID
+- Sign up status
+- Sign up status date
+- Merchant local date
+
+## Optional data
+- User data (user ID, email, etc.)
+- Marketing context data (info about what lead to the sign up attempt)
+- Storefront context data (info about where the user is signing up)
+- Sign up status reason
+
+## Sign up event
+This example request sends a sign up event to Dynamics 365 Fraud Protection asking for a risk assessment of the sign up.
+```http
+POST https://<Merchant API Endpoint>/v1.0/MerchantServices/events/SignUp HTTP/1.1
```

Comment:
nit: Aren't we saying: "https://https://contoso-123.api.dynamics.whatever" here?

Reply:
Yes, thanks!  Fixed to just be <Merchant API Endpoint> everywhere.